openstack/orchestration/v1/stackresources/results.go

```diff
@@ -230,14 +263,6 @@ func (r SchemaResult) Extract() (*TypeSchema, error) {
 	return &res, nil
 }
 
-// TypeTemplate represents a stack resource template.
-type TypeTemplate struct {
```

Comment:
I removed this because the output of http://api.rackspace.com/api-ref-orchestration.html#resource_type_template is a heat template, a string. It doesn't make much sense to parse a heat template, as its considered to be an atomic unit of data for orchestration.